internal/ingress/controller/config/config.go

```diff
@@ -468,6 +468,10 @@ type Configuration struct {
 	// http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive
 	UpstreamKeepaliveConnections int `json:"upstream-keepalive-connections,omitempty"`
 
+	// Sets the maximum time during which requests can be processed through one keepalive connection
+	// https://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive_time
+	UpstreamKeepaliveTime string `json:"upstream-keepalive-time,omitempty"`
```

Comment:
Can we keep the change right now only updating version, and later doing a specific PR for this?

Reply:
@rikatz ,
do you suggest not to add this parameter but just change the Nginx version? Thus, split this PR into two?
PR #1. Increase the Nginx version
PR #2. Introduce the keepalive_time parameter?

Reply:
Yes, exactly l.

Reply:
@strongjz ,

Reply:
@strongjz ,
I removed changes in the go code. Only build.sh scripts that change the Nginx version remained.

I see I should wait for this PR to be merged and then I will create another